src/orchard/misc.clj

```diff
@@ -92,11 +92,10 @@
   (try
     (let [java-ver (System/getProperty "java.version")
           [major minor _] (str/split java-ver #"\.")
-          major (Integer/parseInt major)
-          minor (Integer/parseInt minor)]
+          major (Integer/parseInt major)]
```

Comment:
Might be good to add some docstring that refers to the JEP you mentioned, so people would understand the logic here better.